src/main/resources/templates/checkout.html

```
@@ -1,120 +1,89 @@
+        </div>
+        <div class="info-row">
+          <span class="info-label">결제금액</span>
+          <span class="info-value" id="amountDisplay">—</span>
+        </div>
+        <div class="info-row">
+          <span class="info-label">결제수단</span>
+          <span class="info-value">신용카드 / 체크카드</span>
+        </div>
+      </div>
+
+      <button class="btn-primary" onclick="requestPayment()" id="payBtn">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)